src/app.js

```diff
@@ -0,0 +1,37 @@
+import { productData } from './productData.js';
+import { ProductArray } from './ProducArrary.js';
+const productImageTags = document.querySelectorAll(‘img’);
```

Comment:
lines 3-5 look good but you used back ticks instead of single quotes.

Reply:
no wonder it was still squiggly red